### ISSUE TITLE
[MU4 Issue] Drum Input Palette not fully keyboard accessible

### ISSUE THREAD

**Issue by GabeS573** [musescore/MuseScore#12017]
**Describe the bug**
The Drum Input Palette, which is used to input notes into an unpitched percussion staff, is not accessible by the keyboard alone. The panel cannot be accessed at all with the `tab` key. Since there are only 7 letter keys that can be assigned as shortcuts for percussion instruments, not every instrument in a percussion staff can easily be input by using keyboard alone.

**To Reproduce**
1. Create a score with a drumset part.
2. Select a measure in the drumset part, and press `N` to enter Note Input mode.
3. The drum input palette appears at the bottom of the screen.
4. Try pressing the `tab` key to navigate through the app. The drum palette cannot be accessed by keyboard. While instruments like the Bass Drum that have a letter shortcut (`B` in this case) can easily be input, ones that do not have a shortcut (like the cowbell) cannot.

**Expected behavior**
The drum input palette should be keyboard accessible so that users can input percussion notation and edit a drumset using the keyboard alone.

**Platform information**
 - OS: Win11
 - Private Alpha

**Additional context**
Technically, instruments that don't have a letter shortcut can be input by adding another note that *does* have a shortcut, then using the up/down arrow keys to cycle through the instruments until the desired one is found. However, this method is not very intuitive, and is probably even less intuitive to those who use a screen reader and keyboard navigation...

**Comment by Tantacrul** [comment]
Unless I'm mistaken, this was already an issue with version 3.

We are planning a proper overhaul to the percussion input in 4.x. It is one of our highest priorities. This is a very good observation about another improvement we can make. Thanks!

**Comment by MarcSabatella** [comment]
This came up again on Discord.  Now that the initial 4.0 release is well behind and we're planning 4.x, it might be nice to re-open this issue open to remind us we need to address it in whatever overhaul takes place.

**Comment by Tantacrul** [comment]
Thanks. Reopened

**Comment by Tantacrul** [comment]
Thanks. Reopened

**Comment by GabeS573** [comment]
Since the new percussion panel exists now and is fully accessible, this issue seems moot and I'm going to close it.
